### ISSUE TITLE
Split data set into training and test set

### ISSUE THREAD

**Issue by jannengm** [GVSU-CIS467-Team-Microsoft/Project#4]
From the [Kaggle page](https://www.kaggle.com/c/data-science-bowl-2017/data) containing the data set:

> **stage1_sample_submission.csv** - shows the submission format for stage 1. You should also use this file to determine which patients belong to the leaderboard set of stage 1.

Create a script or other method to split the extracted stage 1 data into a training set and test set as specified by stage1_sample_submission.csv
